spec/features/user_follows_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature "User follows another user" do
+  scenario "User follows and unfollows another user" do
+    leader = Fabricate(:user)
+    comedy = Fabricate(:category)
+    video = Fabricate(:video, category: comedy)
+    review = Fabricate(:review, rating: 5, body: "Great movie", video_id: video.id, user_id: leader.id)
```

Comment:
I think you could use associations to your advantage a bit here - instead of `video_id: video.id, user_id: leader.id` you should be able to pass just `video: video, user: leader` - its less code and a bit more succinct.